spec/enumerable_spec.rb

```diff
@@ -0,0 +1,61 @@
+require './enumerable'
```

Comment:
When I run the `rspec` it gives me an arror `cannot load such file -- ./enumerable`
- Hint: file name is `Enumerable` not `enumerable`